config/manager/kustomization.yaml

```diff
@@ -1,2 +1,7 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: m00nf1sh/misc
```

Comment:
Might want to change the newName field here?

Reply:
sorry, these changes shouldn't be there, it's for follow up changes after this pr merged. I miss pushed to this branch. removed :D